protocol/communities/manager.go

```diff
@@ -173,13 +183,66 @@ func (m *Manager) CreateCommunity(description *protobuf.CommunityDescription) (*
 	return community, nil
 }
 
+// CreateCommunity takes a description, updates the community with the description,
+// saves it and returns it
+func (m *Manager) EditCommunity(request *requests.EditCommunity) (*Community, error) {
+	community, err := m.GetByID(request.CommunityID)
+	if err != nil {
+		return nil, err
+	}
+	if community == nil {
+		return nil, ErrOrgNotFound
+	}
+	if !community.IsAdmin() {
+		return nil, errors.New("not an admin")
+	}
+
+	newDescription, err := request.ToCommunityDescription()
+	if err != nil {
+		return nil, fmt.Errorf("Can't create community description: %v", err)
+	}
+
+	// If permissions weren't explicitly set on original request, use existing ones
+	if newDescription.Permissions.Access == protobuf.CommunityPermissions_UNKNOWN_ACCESS {
+		newDescription.Permissions.Access = community.config.CommunityDescription.Permissions.Access
+	}
+	// If the image wasn't edited, use the existing one
+	// NOTE: This will NOT allow deletion of the community image; it will need to
+	// be handled separately.
+	if request.Image == "" {
+		newDescription.Identity.Images = community.config.CommunityDescription.Identity.Images
+	}
+	// TODO: handle delete image (if needed)
+	newDescription.Clock++
```

Comment:
I don't see clock being copied over/saved in the community, or am I missing something?